src/scheduler/JobScheduler.ts

```diff
@@ -41,7 +41,7 @@ export class JobScheduler {
     const delay = calculateDelay(interval, this.job.immediate, jobEntity);
     this.jobHandle = setIntervalWithDelay(this.executeConcurrently.bind(this), interval, delay);
 
-    this.logger.debug(`scheduled job to run at ${DateTime.fromMillis(delay).toISO()}`, {
+    this.logger.debug(`scheduled job to run at ${DateTime.now().plus(delay).toISO()}`, {
```

Comment:
```suggestion
    this.logger.debug(`scheduled job to run at ${DateTime.now().plus({ millisecond: delay }).toISO()}`, {
```
Could we make this more explicit? I couldn't find the docs for luxon saying the plus function treats numbers as milliseconds.

Reply:
Good point! I'll make the change.